packages/verify/src/verifier.ts

```diff
@@ -43,7 +44,9 @@ export class Verifier {
     this.options = {
       ctlogThreshold: options.ctlogThreshold ?? 1,
       tlogThreshold: options.tlogThreshold ?? 1,
-      tsaThreshold: options.tsaThreshold ?? 0,
+      timestampThreshold:
+        options.timestampThreshold ?? options.tsaThreshold ?? 1,
+      tsaThreshold: 0,
```

Comment:
Is `tsaThreshold` only kept for backward compatibility? 
Look like it is only used here `timestampThreshold: options.timestampThreshold ?? options.tsaThreshold ?? 1,` to support cases that doesn't provide timestampThreshold. 

Can we add a comment for this?